app/serializers/api/v1/address_serializer.rb

```diff
@@ -11,8 +11,8 @@ class AddressSerializer < BaseSerializer
       attribute :street_address_2, &:address2
       attribute :postal_code, &:zipcode
       attribute :locality, &:city
-      attribute :region, &:state_name
-      attribute :country, ->(object, _) { object.country.name }
+      attribute :region, type: :object, &:state_name
+      attribute :country, type: :object, &:country
```

Comment:
This is not quite right. See my comment above. You still put the state name in the region field and the country has too many attributes because it serialises the whole country object. We only want code and name.

Reply:
So we only want to use 'country.iso" here as the 2 letter code right? 👍

Reply:
So what if we specify like:

'{ code = object.state.abbr, name = object.state.name }' and so on..?

Reply:
> So we only want to use "country.iso" here as the 2 letter code right?

Right.

That sounds good.